### ISSUE TITLE
Installation verification shows error

### ISSUE THREAD

**Issue by sunamatya** [HumanCompatibleAI/overcooked_ai#136]
After following the instructions as per https://github.com/HumanCompatibleAI/overcooked_ai,
when verifying the installation was checked, the test fails. 
I have tried installing the program in both windows and ubuntu and the error still persists. 
Attached with this comment is the error.
![failed_overcooked_installation](https://github.com/HumanCompatibleAI/overcooked_ai/assets/54328611/d9d59ea1-e860-467e-afb0-c49903cfcfec)

**Comment by micahcarroll** [comment]
I just [reran the Github actions workflow](https://github.com/HumanCompatibleAI/overcooked_ai/actions/runs/7622113394/job/20759601770) and it seemed to work fine.

I also wasn't able to reproduce the issue locally, which is surprising if you got the same error on two separate machines (is that the case?). In any case, I'm not entirely sure what the issue is caused by without being able to reproduce it.

**Comment by boshenzh** [comment]
@sunamatya Hi, I have the same issue, Were you able to solve it?

**Comment by jyan1999** [comment]
Hi @boshenzh @sunamatya I was able to reproduce this issue on windows, and the problem is the weird default Windows encoding standard not being UTF-8, and when testing we print to a file that contains the upward arrow key, which cannot be recognized by the windows encoding scheme and was causing an error. The permission error is related as we were manually opening and closing the log file before, so when writing failed the file was never closed.

I have pushed a fix to address that. Thx for catching this!
